programs/bpf/rust/get_minimum_delegation_data/src/lib.rs

```diff
@@ -0,0 +1,43 @@
+//! bprumo TODO: doc
+
+#![allow(unreachable_code)]
+
+extern crate solana_program;
+use solana_program::{
+    account_info::AccountInfo,
+    entrypoint::ProgramResult,
+    msg, program,
+    pubkey::Pubkey,
+    stake::{self, MINIMUM_STAKE_DELEGATION},
+};
+
+solana_program::entrypoint!(process_instruction);
+#[allow(clippy::unnecessary_wraps)]
+fn process_instruction(
+    program_id: &Pubkey,
+    accounts: &[AccountInfo],
+    _instruction_data: &[u8],
+) -> ProgramResult {
+    msg!(
+        "get_minimum_delegation_data, program_id: {}, accounts: {:#?}",
+        program_id,
+        accounts
+    );
+
+    //let accounts = &accounts[..];
+    //let accounts = &accounts[1..];
+    let accounts = &accounts[..accounts.len() - 1];
+    msg!("accounts for invoke(): {:#?}", accounts);
+
+    let get_minimum_delegation_instruction = stake::instruction::get_minimum_delegation();
+    let result = program::invoke(&get_minimum_delegation_instruction, accounts);
```

Comment:
`accounts` can be empty because this instruction doesn't need account inputs and the program executable account doesn't need to be passed directly to the invoke. However, the stake program executable account does need to be added to the transaction instruction for the example program so that when the runtime receives the CPI, it can verify that the program was in scope to be invoked. Another way to describe it is that a given program can only invoke other programs if those programs executable accounts were passed as an instruction account input. But again, no need to actually pass the account to the invoke CPI.